src/components/RangeSlider/components/DualThumb/DualThumb.scss

```diff
@@ -133,11 +133,6 @@ $range-output-spacing: rem(16px);
   transition-timing-function: var(--p-ease);
   transform: translateX(calc(-50% + #{$range-thumb-size / 2}));
 
-  .Thumbs:hover + &,
-  .Thumbs:focus + & {
-    opacity: var(--p-override-zero);
-  }
```

Comment:
This seems like it would cause a visual change

Reply:
I didn't see any style changes for this when I tested but would be great to confirm that! Did you get the chance to 🎩 this?

Reply:
Seems ok on 🎩 !